src/services/pseudonym/hooks/index.js

```diff
@@ -42,14 +42,19 @@ const filterValidUsers = (context) => {
 	let validUserIds = [context.params.account.userId];
 	return context.app.service('courses').find({
 		query: {
-			teacherIds: context.params.account.userId,
+			$or: [
+				{ teacherIds: context.params.account.userId },
```

Comment:
what is with `substitutionIds`? asnd because of reuse of `context.params.account.userId` keep that value maybe in something like `currentUserId`?